contracts/upgradeable_contracts/erc20_to_native/HomeBridgeErcToNative.sol

```diff
@@ -0,0 +1,83 @@
+pragma solidity 0.4.24;
+import "../../libraries/SafeMath.sol";
+import "../../libraries/Message.sol";
+import "../BasicBridge.sol";
+import "../../upgradeability/EternalStorage.sol";
+import "../../IBlockReward.sol";
+import "../../ERC677Receiver.sol";
+import "../BasicHomeBridge.sol";
+import "../ERC677Bridge.sol";
+
+
+contract HomeBridgeErcToNative is EternalStorage, BasicBridge, BasicHomeBridge {
+
+    function () public payable {
+        require(msg.value > 0);
+        require(msg.data.length == 0);
+        require(withinLimit(msg.value));
+        setTotalSpentPerDay(getCurrentDay(), totalSpentPerDay(getCurrentDay()).add(msg.value));
+        setTotalBurntCoins(totalBurntCoins().add(msg.value));
+        address(0).transfer(msg.value);
+        emit UserRequestForSignature(msg.sender, msg.value);
+    }
+
+    function initialize (
+        address _validatorContract,
+        uint256 _dailyLimit,
+        uint256 _maxPerTx,
+        uint256 _minPerTx,
+        uint256 _homeGasPrice,
+        uint256 _requiredBlockConfirmations,
+        address _blockReward
+
+    ) public returns(bool)
+    {
+        require(!isInitialized());
+        require(_validatorContract != address(0));
+        require(_homeGasPrice > 0);
```

Comment:
As per specific of the network (Home side of the bridge) this type of the bridge will be used, the bridge validators will not spend any funds for transaction - their addresses will be whitelisted as so transactions from the bridge validators will be allowed with gas price "0".
It means that the fallback gas price which will be set in the Home Bridge contract should be zero. But this check does not allow to set this gas price. Please remove it.